src/redux/selectors/index.js

```diff
@@ -1,6 +1,34 @@
-export const getSearchDetails = state => state.imagesGalleryPage
+import { createSelector } from 'reselect'
+
+import { ITEMS_PER_PAGES } from 'common/constants'
+
+// General
+export const getIsLoading = state => state.imgDailyPage.isLoading
+
+//  Image gallery
+export const getImages = state => state.imagesGalleryPage.items
+
+export const getImagesSelector = createSelector(
+  getImages,
+  items => items.flatMap(({ links }) => links),
+)
 
+// Daily image
+export const getDailyImage = state => state.imgDailyPage.image
```

Comment:
```
export const getDailyImage = state => state.imgDailyPage.image
export const getIsLoading = state => state.imgDailyPage.isLoading
```

**state.imgDailyPage** is 2 times destructured, it should be one time


thats why you should have always smaller selectors, one to get you **imgDailyPage** from state, and 2 others to get you  **image** and **isLoading** props from that